go.mod

```diff
@@ -92,11 +94,9 @@ require (
 	github.com/hashicorp/hcl v1.0.0 // indirect
 	github.com/holiman/bloomfilter/v2 v2.0.3 // indirect
 	github.com/holiman/uint256 v1.2.4 // indirect
-	github.com/huandu/go-clone v1.6.0 // indirect
-	github.com/huandu/go-clone/generic v1.7.2 // indirect
 	github.com/inconshreveable/mousetrap v1.1.0 // indirect
 	github.com/ipfs/bbloom v0.0.4 // indirect
-	github.com/ipfs/boxo v0.17.1-0.20240131173518-89bceff34bf1 // indirect
+	github.com/ipfs/boxo v0.19.0 // indirect
```

Comment:
This shouldn't be updated, the old version is the correct one. See https://github.com/onflow/flow-go/pull/5822. The update to 0.19 is causing the need to update to Go 1.21